content/docs/setup/kubernetes/helm-install.md

```diff
@@ -111,6 +112,38 @@ The Helm chart also offers significant customization options per individual
 service. Customize these per-service options at your own risk. The per-service options are exposed via
 the [`values.yaml`](https://raw.githubusercontent.com/istio/istio/{{<branch_name>}}/install/kubernetes/helm/istio/values.yaml) file.
 
+## Customization example: traffic management minimal set
+
+Istio is equipped with a rich and powerful set of features and some users may need only subset of those. For instance, users might be interested only
+in installing the minimal set required to Istio's traffic management.
+[Helm customization](#customization-with-helm) provides the option to install a subset by enabling those features of interest and disabling the ones that aren't required.
+
+In this example we will install Istio with only a minimal set of components necessary to conduct [traffic management](/docs/tasks/traffic-management/).
+
+Execute the following command to install the Pilot, Citadel, IngressGateway and Sidecar-Injector:
+
+```command
+$ helm install install/kubernetes/helm/istio --name istio --namespace istio-system \
+  --set ingress.enabled=false,egressgateway.enabled=false,galley.enabled=false \
+  --set mixer.enabled=false,prometheus.enabled=false,global.proxy.envoyStatsd.enabled=false
+```
+
+Ensure the following Kubernetes pods are deployed and their containers are up and running: `istio-pilot-*`, `istio-ingressgateway-*`,
+`istio-citadel-*` and `istio-sidecar-injector-*`.
```

Comment:
Aren’t these two completely unnecessary for pilot plus gateway?

Reply:
Pilot's bootstrap strictly needs certs as of yesterday. I had noticed that certs became a pre-requisite at some point, so citadel is a requirement.

Reply:
@rshriram, @kyessenov is right. I also don't like the tight dependency and would prefer an option to install Pilot and use the default secret that k8s injects. While possible to set Pilot to take other certs path with an arg it's more difficult with envoy.

As for sidecar-injector, I mentioned later on that user can omit this too for manual injection.